client/me/memberships/subscription.jsx

```diff
@@ -33,42 +33,67 @@ function Subscription( { translate, subscription, moment, stoppingStatus } ) {
 	useEffect( () => {
 		if ( stoppingStatus === 'fail' ) {
 			// run is-error notice to contact support
-			dispatch(
-				errorNotice(
-					translate(
-						'There was a problem while stopping your subscription, please {{a}}{{strong}}contact support{{/strong}}{{/a}}.',
-						{
-							components: {
-								a: <a href={ CALYPSO_CONTACT } />,
-								strong: <strong />,
-							},
-						}
-					)
-				)
-			);
+			subscription && ! subscription.renew_interval
```

Comment:
Rather than duplicating the `subscription && ! subscription.renew_interval` check across multiple place, I'm wonder if we should use a variable with a better name:

```js
const isProduct = subscription && ! subscription.renew_interval;
```

Then we could replace all the checks with that var which reads better?